src/rcore.c

```diff
@@ -1718,6 +1718,9 @@ int GetCurrentMonitor(void)
         int x = 0;
         int y = 0;
 
+        int best_overlap = 0;
```

Comment:
Please, could you review those variables naming to follow [raylib coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md).